### ISSUE TITLE
Redsea Keeps asking for username and password

### ISSUE THREAD

**Issue by SebTheHackerman** [redsudo/RedSea#60]
![image](https://user-images.githubusercontent.com/63127804/78406915-b2680e80-75d1-11ea-82a4-25127998b3eb.png)
It keeps asking me for a username and password even after I name the session. Someone pls help Idk what to do to fix this thanks.

**Comment by hyacin75** [comment]
Yeah, I'm getting the same.  I'm also curious about the readme saying "It has been deprecated in favor of a more functional replacement. " (or something to that effect) - what the "more functional replacement" is?

Update: This is working, I'm just going to use it for now - https://github.com/yaronzz/Tidal-Media-Downloader

**Comment by SebTheHackerman** [comment]
Alright thanks

**Comment by alexandrox** [comment]
Try this https://github.com/redsudo/RedSea/issues/57 and use this token: 

> old token = 'u5qPNNYIbD0S0o36MrAiFZ56K6qMCrCmYPzZuTnV'
> new token = 'MbjR4DLXz1ghC4rV'

In line 152 change the token by:

> 'hZ9wuySZCmpLLiui'

**Comment by Bredkov** [comment]
Yes. New token helped. Works well.

**Comment by moonraker2** [comment]
+1 for [Tidal-Media-Downloader / tidal-dl](https://github.com/yaronzz/Tidal-Media-Downloader) as a better alternative to Redsea.

**Comment by pintyoke** [comment]
Unfortunately I don't go for the FLAC download. Only m4a! Do you have it too?

Storno! I didn't rewrite it...

**Comment by alexandrox** [comment]
> 
> 
> Unfortunately I don't go for the FLAC download. Only m4a! Do you have it too?
> 
> Storno! I didn't rewrite it...

It's the token, change the token and login account again with the new token.

No mobile token:
>MbjR4DLXz1ghC4rV
